libraries/Microsoft.Bot.Connector/OAuthClientConfig.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Microsoft.Rest;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Connector
+{
+    public static class OAuthClientConfig
+    {        
+        /// <summary>
+        /// The default endpoint that is used for API requests.
+        /// </summary>
+        public static string OAuthEndpoint { get; set; } = AuthenticationConstants.OAuthUrl;
+        
+        /// <summary>
+        /// When using the Emulator, whether to emulate the OAuthCard behavior or use connected flows
+        /// </summary>
+        public static bool EmulateOAuthCards { get; set; } = false;
+
+        /// <summary>
+        /// Send a dummy OAuth card when the bot is being used on the Emulator for testing without fetching a real token.
+        /// </summary>
+        /// <param name="emulateOAuthCards">Indicates whether the Emulator should emulate the OAuth card.</param>
+        /// <returns>A task that represents the work queued to execute.</returns>
+        public static async Task SendEmulateOAuthCardsAsync(OAuthClient client, bool emulateOAuthCards)
```

Comment:
This method should take a Cancellation Token as an optional argument. Default that argument to "CancellationToken.None".